### PR TITLE
fix: delete redundant .media + .media

### PR DESCRIPTION
**Pull request by dozortsev**
## 🚀 Changes

See comments below with description.

## 🤔 Considerations

All React components which somehow uses `.media` class are not affected:

- [Accordion](https://transferwise.github.io/neptune-web/branch/delete-media/components/content/Accordion)
- [Card](https://transferwise.github.io/neptune-web/branch/delete-media/components/content/Card)
- [CheckboxOption](https://transferwise.github.io/neptune-web/branch/delete-media/components/content/CheckboxOption)

## ✅ Checklist

- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/css/src/less/media.less [hunk 1]
```diff
@@ -8,10 +8,6 @@
   display: flex;
 }
 
-.media + .media {
-  margin-top: calc(var(--spacer) * 3);
-}
```
Comment (dozortsev) [inline packages/css/src/less/media.less:13]: this fix removes gap, for context: https://github.com/transferwise/verification-components/pull/233#issuecomment-671851585

Screen Shot 2020-08-12 at 12 12 55 PM

**Comment by helendurranttw** [comment]
I'm a bit concerned we might break other instances of people using components with .media. Have you looked through styleguide-components to see where they use media in other components?

**Comment by dozortsev** [comment]
@helendurranttw just have done, there are fine, those two components use `.media` class, here is them using `neptune.css` with out the rule:

Screen Shot 2020-08-12 at 1 44 56 PM

Screen Shot 2020-08-12 at 1 44 21 PM

here is with the rule, so as you can see it's not even gets applied:

Screen Shot 2020-08-12 at 1 53 07 PM

**Comment by StevePole** [comment]
I can't think of a reason for this rule to exist.  Generally speaking media elements should be contiguous.  If there is a case where they should not be, that should be the exception and there should be a special rule to provide space in that case.

**Pull request merged**